index.js

```diff
@@ -151,3 +139,30 @@ function reportLocationUpdateError(reason, callback) {
         })
     });
 }
+
+module.exports = {
```

Comment:
This `module.exports` section I have not seen before. @Tang-1996 Did you follow a tutorial that told you to do this? If so, please can you reply with a link?